packages/grafana-ui/src/components/FormLabel/FormLabel.tsx

```diff
@@ -10,7 +10,7 @@ interface Props {
   isFocused?: boolean;
   isInvalid?: boolean;
   tooltip?: PopoverContent;
-  width?: number;
+  width?: number | string;
```

Comment:
I think we can just use `width.toString()` later on if that is the issue or can this really be something else than number?

Reply:
I wanted to be able to supply "auto" as a value for the `width` prop so the `width-auto` class would be used. Could change this so that rather than the type being `number | string` it could be `number | "auto"` though

Reply:
Yep `number | "auto"` would definitely makes sense here.